src/shared/components/challenge-detail/Header/TabSelector/index.jsx

```diff
@@ -77,11 +77,16 @@ export default function ChallengeViewSelector(props) {
           styleName={getSelectorStyle(selectedView, DETAIL_TABS.DETAILS, trackLower)}
         >DETAILS
         </a>
-        <a
-          onClick={(e) => { handleSelectorClicked(e, DETAIL_TABS.REGISTRANTS); }}
-          styleName={getSelectorStyle(selectedView, DETAIL_TABS.REGISTRANTS, trackLower)}
-        >REGISTRANTS {numRegistrants ? `(${numRegistrants})` : ''}
-        </a>
+        {
+          numRegistrants > 0 &&
+          <a
+            onClick={(e) => {
+              handleSelectorClicked(e, DETAIL_TABS.REGISTRANTS);
+            }}
+            styleName={getSelectorStyle(selectedView, DETAIL_TABS.REGISTRANTS, trackLower)}
+          >REGISTRANTS {numRegistrants ? `(${numRegistrants})` : ''}
```

Comment:
As the entire block is rendered conditionally now, no need for `{numRegistrants ? `(${numRegistrants})` : ''}`, should be just `({numRegistrants})`